SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Eiffel-community vulnerability and security reporting and response
+
+## Report a Vulnerability or Security issue
+We as a community encourage researchers, users and contributors to report vulnerabilities and security related issues to the Eiffel community. All issues are thoroughly investigated by a community security officer and/or other community security volunteers. All reported and fixed security and vulnerability issues can be found on the [Eiffel community security page](https://eiffel-community.github.io/security.html) .
+
+## How to report a security vulnerability
```

Comment:
Should probably be Title Case in the header. I.E. "How to Report a Security Vulnerability"

Reply:
fixed